### PR TITLE
refactor: remove checking URL and add more form inputs

### PR DESCRIPTION
**Pull request by agatha197**
**Changes:**

This PR updates the ImportFromHuggingFaceModal component and related translations:

- Simplified the form structure by removing the separate check button and combining URL validation with form submission
- Added new form fields for "Model store folder name" and "Service name"
- Updated URL validation to occur on form submission rather than a separate check step
- Removed unused state variables and effects
- Updated translations for all supported languages to include new field labels

**Rationale:**

These changes streamline the user experience when importing models from Hugging Face by reducing the number of steps required. The additional form fields allow for more detailed model information to be captured during the import process.

**Effects:**

- Users will now validate and submit the Hugging Face URL in a single step
- Users can specify a model store folder name and service name when importing
- The UI is cleaner and more straightforward, potentially reducing user confusion

**Screenshots:**
- default

- valid URL

- invalid URL

**Checklist:**

- [ ] Documentation: Update user guide to reflect new import process and additional fields
- [ ] Test case: Verify that URL validation occurs correctly on form submission
- [ ] Test case: Ensure new fields (folder name and service name) are properly saved and utilized

### CODE REVIEW

**Comment by agatha197** [comment]
> [!WARNING]
> <b>This pull request is not mergeable via GitHub because a downstack PR is open. Once all requirements are satisfied, merge this PR as a stack <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2714?utm_source=stack-comment-downstack-mergeability-warning" >on Graphite</a>.</b>
> <a href="https://graphite.dev/docs/merge-pull-requests">Learn more</a>

* **#2713** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2713?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2714** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2714?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* **#2710** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2710?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2709** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2709?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* **#2514** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2514?utm_source=stack-comment-icon" target="_blank">Graphite</a>
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @agatha197 and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/ImportFromHuggingFaceModal.tsx [hunk 1]
```diff
@@ -39,7 +36,6 @@ const ImportFromHuggingFaceModal: React.FC<ImportFromHuggingFaceModalProps> = ({
   const formRef = useRef<FormInstance<Service>>(null);
   const [isImportOnly, { toggle: toggleIsImportOnly }] = useToggle(false);
   const [huggingFaceURL, setHuggingFaceURL] = useState<string | undefined>();
```
Comment (yomybaby) [inline react/src/components/ImportFromHuggingFaceModal.tsx:38]: After removing URL checking, we can remove this.

**Review by yomybaby — CHANGES_REQUESTED** [review]
After removing URL checking, we can remove more code related to URL checking such as `huggingFaceURL` state,  `isPendingCheck` transition, `hugginFaceModelInfo` useUspenseTanQuery.

**Review by yomybaby — APPROVED** [review]
LGTM

**Review by agatha197 — COMMENTED** [review]
handled in https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2715/feat-Improve-form-item-and-switch-to-README-preview-on-validation-check

**Pull request closed without merging**